deploy/restartComponent.sh

```diff
@@ -8,7 +8,11 @@
 
 HOST=$(hostname)
 DATENOW=$(date +%s)
-DEST_NAME=cms-wmcore-team
+# look-up alert emails from WMA secret file where EMAIL_ALERT may contains multiple emails and spaces, e.g.
+# EMAIL_ALERT="user1@domain.com user2@domain.com" or EMAIL_ALERT = "user1@domain.com user2@domain.com"
+EMAIL_ALERT=`sed -E 's/^[[:space:]]*EMAIL_ALERT[[:space:]]*=[[:space:]]*"([^"]*)".*/\1/' $WMA_SECRETS_FILE`
+
+[[ -z $EMAIL_ALERT ]] && {echo "ERROR: unable to find EMAIL_ALERT in $WMA_SECRETS_FILE"; exit 1;}
```

Comment:
As reported above, you mistyped the configuration name attribute.

Reply:
thanks for pointing this out, indeed my mistake. In fact, it is good that you point to this as I think it is much more natural to use `ALERT_EMAILS` instead of the current `EMAIL_ALERTS`. To me former is better represent what it is. So, I changed the code appropriately and changed WMAgent configs accordingly.

Reply:
ALERT_EMAILS looks much better indeed! Thanks